### DISCUSSION TITLE
Criar Database no Dapper ou no SQL Server

### DISCUSSION THREAD

**Discussion by pedro-mileipp** [balta-io/forum#191] [Tira Dúvidas] [answered]
- Boa tarde, prezado! Como vai? 
- Para criar a Database é mais recomendado que seja no Dapper ou no SQL Server?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @pedro-mileipp, como vai?

O Dapper é apenas uma extensão do pacote Microsoft.Data.SqlClient, que provê acesso aos dados no SQL Server.
Desta forma, o lugar correto para criar um Database é sim no SQL Server.

Além disso, existem pacotes como o Dapper.Migrations e Entity Framework que **geram** o banco de dados para nós.
Nós tratamos isto na parte de migrações, no curso de EF.
